test/E2ETest/TyeInitTests.cs

```diff
@@ -74,5 +74,24 @@ public void FrontendBackendTest()
 
             Assert.Equal(expectedContent.NormalizeNewLines(), content.NormalizeNewLines());
         }
+
+        // Tests our logic that excludes non-applications (unit tests, classlibs, etc)
+        [Fact]
+        public void Init_ProjectKinds()
+        {
+            using var projectDirectory = CopyTestProjectDirectory("project-types");
```

Comment:
Do you think having a tye run test is necessary here to verify that the application doesn't run any test projects? I don't think it does, but just wanted to pick your brain.

Reply:
I like where your head is at, but I don't really think so. If this were a shipping product maybe 👍 I'm optimizing for current/future productivity and the risk of a regression is low.